### DISCUSSION TITLE
The branch in the pull request is different from the branch stored on the github

### DISCUSSION THREAD

**Discussion by phellka** [community/community#22527] [New to GitHub]
<p>I have three braches. Emty main brach, lab1_base and lab1_dop. File in lab1_base and lab1_dop looks the same.<br>
But if I create pull request compare files looks like:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/f/2f04c474a5d72d4ebfe562d7e20a1815978d1557.png" data-download-href="/uploads/short-url/6HWECHpX0LXLva0aUBbB23NgVy7.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/90329682/181071631-2c43df5f-4242-4aa4-9897-b3f949aa0971.png" alt="image" data-base62-sha1="6HWECHpX0LXLva0aUBbB23NgVy7" width="690" height="203" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/f/2f04c474a5d72d4ebfe562d7e20a1815978d1557_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1249×369 21.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>How i can compare the lastest versions of branches,  that stored in github,  in pull request?</p>

**Comment by phellka** [comment]
<p>I think i have diferent version of file in pull request, what havent all commits. But i anyway don’t know how compare newest versions.</p>

**Comment by jfullstackdev** [comment]
<p>you mean, when you created the pull request, after that, that branch is outdated already? so there is the confusion?</p>

**Comment by phellka** [comment]
<p>when I view the contents of the files in the branches, they have the same contents. But when I want to make a pull request it shows different content. I don’t execute pull request because it shows not the actual contents of the file</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>am I right, this is the repo?</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/phellka/PIbd-21-Madyshev-E.A.-Car-Factory">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/phellka/PIbd-21-Madyshev-E.A.-Car-Factory" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181071638-6c7c0520-1cf2-4c2b-a512-12e32e5e13dc.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/2/524674890f53ebb817e594967a6e286ef5d5b261_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/2/524674890f53ebb817e594967a6e286ef5d5b261_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/2/524674890f53ebb817e594967a6e286ef5d5b261.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/2/524674890f53ebb817e594967a6e286ef5d5b261_2_10x10.png"></div>

<h3><a href="https://github.com/phellka/PIbd-21-Madyshev-E.A.-Car-Factory" target="_blank" rel="noopener">GitHub - phellka/PIbd-21-Madyshev-E.A.-Car-Factory</a></h3>

  <p>Contribute to phellka/PIbd-21-Madyshev-E.A.-Car-Factory development by creating an account on GitHub.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<aside class="quote no-group" data-username="phellka" data-post="4" data-topic="228875">
<div class="title">
<div class="quote-controls"></div>
 phellka:</div>
<blockquote>
<p>when I view the contents of the files in the branches, they have the same contents</p>
</blockquote>
</aside>
<p>they are not the same, just by simply comparing them, you will see it, just like this,</p>
<p>click the link to see it,</p>
<p><a href="https://github.com/phellka/PIbd-21-Madyshev-E.A.-Car-Factory/compare/lab1_dop...lab1_base">https://github.com/phellka/PIbd-21-Madyshev-E.A.-Car-Factory/compare/lab1_dop...lab1_base</a></p>
